README.md

```diff
@@ -90,6 +91,85 @@ private void ConfigureNetworkExceptionHandler(IServiceCollection services)
 }
 ```
 
+### SimpleCacheHandler
+
+The `SimpleCacheHandler` is a `DelegatingHandler` that executes custom caching instructions.
+
+When you use Refit for your endpoints declaration, you can neatly specify caching instructions with attributes.
+
+- You can specify time-to-live at different levels.
+  - On a per-call level (using attributes)
+  - Globally (using default headers)
+- You can support force-refresh scenarios (i.e. don't read the cache, but update it).
+This can be useful for things like pull-to-refresh.
+- You can disable the cache on a per-call level.
+This only makes sense when you define default caching globally.
+
+```csharp
+// (...)
+public static class CacheInstructions
+{
+  public const string DefaultCacheValue = "600"; // 10 minutes
```

Comment:
it's a shame that we have to setup those values by append to string directly. I think that ideally we should either have a config class (or even if we could parametrize this in appsettings.json per environment).

Reply:
Oh you **can** leverage `IOption` to setup a default cache ttl if you want.
We use strings directly because we go through HTTP headers.
We use HTTP headers because we want to be able to customize the values at the declaration of a Refit endpoint.